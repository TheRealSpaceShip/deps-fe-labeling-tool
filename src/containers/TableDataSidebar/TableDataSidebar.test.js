import React from 'react'
import { mockComponent } from '@/mocks/mockComponent'
import { shallow } from 'enzyme'
import { TableDataSidebar } from '@/containers/TableDataSidebar'

jest.mock('@/containers/TableData', () => mockComponent('TableData'))

describe('Component: TableDataSidebar', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<TableDataSidebar />)
  })

  it('should render correct layout', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
