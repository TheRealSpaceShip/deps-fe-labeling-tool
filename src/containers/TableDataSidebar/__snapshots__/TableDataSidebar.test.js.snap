// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: TableDataSidebar should render correct layout 1`] = `
<TableDataSidebarstyles__TabsStyled
  defaultActiveKey="Table Data"
>
  <DeprecatedTabPane
    key="Table Data"
    tab="Table Data"
  >
    <TableData />
  </DeprecatedTabPane>
</TableDataSidebarstyles__TabsStyled>
`;
